netbox/netbox/graphql/schema.py

```diff
@@ -38,7 +38,8 @@ class Query(
     query=Query,
     config=StrawberryConfig(auto_camel_case=False),
     extensions=[
-        DjangoOptimizerExtension(prefetch_custom_queryset=True),
+        # DjangoOptimizerExtension(prefetch_custom_queryset=True),
```

Comment:
```suggestion
```

Reply:
If we get Strawberry-Django updated to 0.55.1 it will fix #18157 with the optimizer still enabled.

Reply:
Went ahead and enabled the optimizer